openedx/core/djangoapps/user_api/accounts/api.py

```diff
@@ -243,17 +243,21 @@ def _validate_name_change(user_profile, data, field_errors):
         return None
 
     old_name = user_profile.name
+    new_name = data['name']
+
+    if old_name == new_name:
```

Comment:
You may want to update the account MFE code that works around this or log a ticket to do that. I added code to send the `full_name` on all requests from IDV, but only when the verified name feature is enabled in [this PR](https://github.com/edx/frontend-app-account/pull/508). This was to prevent a name that hasn't changed from being saved in the `old_names` field of the profile metadata. I don't think that will be necessary anymore, given this change.